storybook/src/components/card/card.core-template.ts

```diff
@@ -53,13 +60,10 @@ export const coreCard: ComponentImplementation<Card<never>> = {
             ${interactions.map(
               (interaction) => html`
                 <div class="dso-card-interaction">
-                  ${isButtonInterface(interaction)
-                    ? buttonTemplate(interaction)
-                    : isLabelInterface(interaction)
-                      ? labelTemplate(interaction)
-                      : isToggletipInterface(interaction)
-                        ? toggletipTemplate(interaction)
-                        : nothing}
+                  ${isButtonInterface(interaction) ? buttonTemplate(interaction) : nothing}
+                  ${isLabelInterface(interaction) ? labelTemplate(interaction) : nothing}
+                  ${isToggletipInterface(interaction) ? toggletipTemplate(interaction) : nothing}
+                  ${isSlideToggleInterface(interaction) ? slideToggleTemplate(interaction) : nothing}
```

Comment:
Hehe, leuk. Ik begrijp het, maar realiseer dat de gerenderde markup mbt. whitespaces anders is:

```html
<!-- vorige situatie, met nested ternaries -->
<div class="dso-card-interaction">
  <dso-slide-toggle></dso-slide-toggle>
</div>

<!-- nieuwe situatie, zonder nested ternaries -->
<div class="dso-card-interaction">
  <!--?lit$750772243$-->
  <!--?lit$750772243$-->
  <!--?lit$750772243$-->
  <!--?lit$750772243$-->
  <dso-slide-toggle></dso-slide-toggle>
</div>
```

`nothing` geeft geen whitespace terug, maar het zijn de whitespaces buiten de expressies (`${}`).

Voor "Card with label" ziet het er bv. zo uit:

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/80f6e2a6-94fa-41a8-b95f-f2b7087f6600)

![image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/ad90b27d-470e-4347-af7e-bb261616a260)

Inhoudelijk maakt het niet zoveel uit. Ik ben ook geen fan van de nested ternary, maar het had wel 'n doel.

Het is aan jou!

Reply:
nested ternary gehandhaaft.